website/docs/r/cloud_external_network.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+subcategory: "Cloud"
+layout: "aci"
+page_title: "ACI: aci_cloud_external_network"
+sidebar_current: "docs-aci-resource-aci_cloud_external_network"
+description: |-
+  Manages ACI Cloud External Network
+---
+
+# aci_cloud_external_network #
+
+Manages ACI Cloud External Network.
+<b>Note: This resource is supported in Cloud APIC version > 25.0 only.</b>
+
+## API Information ##
+
+* `Class` - cloudtemplateExtNetwork
+* `Distinguished Name` - uni/tn-{tenant_name}/infranetwork-{name}/extnetwork-{name}
+
+## GUI Information ##
+
+* `Location` - Tenants -> {tenant_name} -> Application Management -> External Networks
```

Comment:
I couldn’t find the `External Networks` under the `Tenants -> Application Management`.